website/source/docs/post-processors/vagrant.html.md

```diff
@@ -102,7 +103,7 @@ In the example above, the compression level will be set to 1 except for VMware,
 where it will be set to 0.
 
 The available provider names are: `aws`, `digitalocean`, `google`, `virtualbox`,
-`vmware`, and `parallels`.
+`vmware`, `parallels` and `lxc`.
```

Comment:
Done

Reply:
A nitpick, but there should be a coma before the and.

Reply:
Swapped to a list and added the missing providers